x-pack/legacy/plugins/alerting_ui/public/index.scss

```diff
@@ -0,0 +1,8 @@
+// Import the EUI global scope so we can use EUI constants
+@import 'src/legacy/ui/public/styles/_styling_constants';
+
+// Alerting UI plugin styles
+
+.alertingUIActionContextMenuItem {
```

Comment:
Rather than using this, just apply `alignItems="center"` to the `EuiFlexGroup`

Reply:
Fixed, as you proposed.